### DISCUSSION TITLE
Set attribute src dynamically from input file

### DISCUSSION THREAD

**Discussion by Aubi0ne** [google/model-viewer#2049] [Q&A] [answered]
Hi,

I am currently creating a gltf file upload form and would like to know: is it possible to view the gltf file the user selects from the input file by updating the src attribute of the model-viewer?
Because src attribute need gltf file path and input file just contain the name of the selected gltf ^^"

**Comment by Aubi0ne** [comment, marked as answer]
Ok can create a url with URL.createObjectURL(e.target.files[0]) and it works nice :)

**Comment by Tonikuu** [comment]
Hi, can you explain how you did it? I have same need. I have file input where I can upload multiple files, like gltf model needs, but can't figure out how to put it to model-viewer src.
